docs/content/getting-started/setup-openfga.mdx

```diff
@@ -295,13 +295,31 @@ The following table table enumerates the experimental flags, a description of wh
 | name                   | description                                                                                              | openfga version |
 |------------------------|----------------------------------------------------------------------------------------------------------|-----------------|
 | list-objects-optimized | Enables support for an optimized version of the [openfga.v1.ListObjects](/api/service#/Relationship%20Queries/ListObjects) API for models with schema version 1.1 | >= v0.3.1       |
+| otel-metrics           | Enables support for exposing OpenFGA metrics through OpenTelemetry | >= v0.3.2       |
 
 :::caution Warning
 Experimental features are not guaranteed to be stable and may lead to server instabilities. It is not recommended to enable experimental features for anything other than experimentation.
 
 Experimental feature flags are also not considered part of API compatibility and are subject to change, so please refer to each OpenFGA specific release for a list of the experimental feature flags that can be enabled for that release.
 :::
 
+## Telemetry (Metrics and Tracing)
```

Comment:
Is this supposed to be one level under experimental features? If yes, let's add one more `#`. If not, can you add a quick not that you need to pass in `--experimentals="otel-metrics"` below? Because right now the two sections seem disconnected and it reads like the two otel flags (`--otel-telemetry-endpoint` and `--otel-telemetry-protocol`) are enough

Reply:
Telemetry is a second-level header like "Experimental Features" and the other sections above it. This section can (and should) evolve independent of the Experimental Features section). They are not mean to flow together.

The Metrics sub-section herein explicitly mentions the necessity to enable the `otel-metrics` experimental feature, and provides a link up to the Experimental Features section. That was purposefully done to make this section independent and unique. In the future when metrics become a first-class citizen and aren't an experimental feature, this section becomes unrelate to experimental features altogether.

Reply:
Are you just saying to provide an example of the `openfga run --experimentals=otel-metrics ...` command?

Reply:
Yep, just a quick example on line 319 would be very helpful

Reply:
👍 let me know how that looks. I changed the wording a bit and added an example.

Reply:
Looks 👍🏽  to me!